cargo-cyclonedx/src/cli.rs

```diff
@@ -64,6 +67,14 @@ pub struct Args {
         value_name = "FILENAME_PREFIX"
     )]
     pub output_prefix: Option<String>,
+
+    /// Perform a best-effort interpretation of invalid SPDX license expressions (lax parsing)
```

Comment:
```suggestion
    /// Accept the deprecated '/' separator for licenses, treating 'MIT/Apache-2.0' as 'MIT OR Apache-2.0'
```

I think it would be much more helpful this way.